services/orders_create/lambda_function.py

```diff
@@ -37,13 +25,13 @@ def lambda_handler(event, context):
                 'total_cost': decimal.Decimal('0.0')
             }
         )
-        res = str(json.dumps(response, cls=DecimalEncoder))
+        res = str(json.dumps(response, default=str))
```

Comment:
Good find! :)

Reply:
thanks, indeed it simplifies things quite a bit 😄